docs/tool/js/views/chloropleth-legend.js

```diff
@@ -36,11 +36,9 @@ var chloroplethLegend = {
           return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","); //adds thousands separators, from https://stackoverflow.com/questions/2901102/how-to-print-a-number-with-commas-as-thousands-separators-in-javascript
```

Comment:
Again, using d3.format() would be cleaner - not blocking this PR unless you want to change it now, but noted for future refactoring.